contracts/anchor_basset_hub/src/contract.rs

```diff
@@ -355,9 +355,7 @@ fn withdraw_all_rewards(deps: &DepsMut, delegator: String) -> StdResult<Vec<Cosm
     Ok(messages)
 }
 
-/// Check whether slashing has happened
-/// This is used for checking slashing while bonding or unbonding
-pub fn slashing(deps: &mut DepsMut, env: Env) -> StdResult<State> {
+fn update_state(deps: Deps, env: Env) -> StdResult<State> {
```

Comment:
update_state here is misleading (no updates actually happen). We should rename this